### DISCUSSION TITLE
Feature request: RadioGroup disabled state

### DISCUSSION THREAD

**Discussion by mikestecker** [tailwindlabs/headlessui#346] [Ideas]
I see there is a `disabled` prop for RadioGroup in the React version, however it doesn't do anything. Can you add this feature?

**Comment by henribru** [comment]
It would be great to have working `disabled` props for all the input components. Currently there's one for Listbox and ListboxOption, but not RadioGroupOption and Switch. In https://github.com/tailwindlabs/headlessui/issues/112 it was mentioned that it would be good to be consistent on this point, which I definitely agree with.

Edit: Seems we're getting one for the radio components! https://github.com/tailwindlabs/headlessui/pull/401

**Comment by Cherry** [comment]
I'd love to see a consistent `disabled` property across all of these components too. `Switch` in particular is the key one we're missing today.
